### PR TITLE
3961-Notes-Relation-Field

### LINKED ISSUE #3961: Notes relation field should be aligned on left
# Current behavior

The relation field is centered

CleanShot 2024-02-14 at 11 30 01

# Desired Behavior

The relations should be aligned on the left side

Screenshot 2024-02-14 at 11 30 56

### PR DESCRIPTION
**Pull request by Kanav-Arora**
Closes #3961 

Screenshot 2024-02-14 at 6 58 11 PM

### CODE REVIEW

File: packages/twenty-front/src/modules/object-record/record-inline-cell/components/RecordInlineCellContainer.tsx [hunk 1]
```diff
@@ -62,7 +62,7 @@ const StyledClickableContainer = styled.div`
 const StyledInlineCellBaseContainer = styled.div`
   align-items: center;
   box-sizing: border-box;
-
+  width: 100%;
```
Comment (FelixMalfait) [inline packages/twenty-front/src/modules/object-record/record-inline-cell/components/RecordInlineCellContainer.tsx:65]: Thanks for the PR! This broke the centering for other elements (look at the Title on the record of a show page for example)

File: packages/twenty-front/src/modules/activities/inline-cell/components/ActivityTargetsInlineCell.tsx [hunk 2]
```diff
@@ -39,13 +39,14 @@ export const ActivityTargetsInlineCell = ({
...
         IconLabel={IconArrowUpRight}
+        showLabel={true}
         editModeContent={
           <ActivityTargetInlineCellEditMode
             activity={activity}
             activityTargetObjectRecords={activityTargetObjectRecords}
           />
         }
-        label="Relations"
+        label="Relations:"
```
Comment (charlesBochet) [inline packages/twenty-front/src/modules/activities/inline-cell/components/ActivityTargetsInlineCell.tsx:49]: should we add the semi column? It looks like we don't have it for other fields

File: packages/twenty-front/src/modules/object-record/record-inline-cell/components/RecordInlineCellContainer.tsx [hunk 3]
```diff
@@ -160,7 +160,7 @@ export const RecordInlineCellContainer = ({
               content={label}
               clickable
               noArrow
-              place="left"
+              place="bottom"
```
Comment (charlesBochet) [inline packages/twenty-front/src/modules/object-record/record-inline-cell/components/RecordInlineCellContainer.tsx:163]: This change seems unrelated to the PR
@Bonapara what do you think should the tool tip be below or on left of the hoverable label?

File: packages/twenty-front/src/modules/activities/inline-cell/components/ActivityTargetsInlineCell.tsx (hunk 2 above)
Comment (FelixMalfait) [reply packages/twenty-front/src/modules/activities/inline-cell/components/ActivityTargetsInlineCell.tsx:49]: cc @Bonapara since it was like that on his design

Comment (Bonapara) [reply packages/twenty-front/src/modules/activities/inline-cell/components/ActivityTargetsInlineCell.tsx:49]: Yes well noticed, we can get rid of the semicolon!

File: packages/twenty-front/src/modules/object-record/record-inline-cell/components/RecordInlineCellContainer.tsx (hunk 3 above)
Comment (Bonapara) [reply packages/twenty-front/src/modules/object-record/record-inline-cell/components/RecordInlineCellContainer.tsx:163]: No strong opinion, but maybe below is more responsive?

**Comment by charlesBochet** [comment]
@Kanav-Arora could you make the small changes? Then we are good to go!

**Comment by Kanav-Arora** [comment]
Hi @charlesBochet 
Missed this what changes?

File: packages/twenty-front/src/modules/object-record/record-inline-cell/components/RecordInlineCellContainer.tsx (hunk 3 above)
Comment (Kanav-Arora) [reply packages/twenty-front/src/modules/object-record/record-inline-cell/components/RecordInlineCellContainer.tsx:163]: Actually for left the tool tip was overlaying sidebar's text

**Review by charlesBochet — APPROVED** [review]
We are good to go @Kanav-Arora!
